src/rdkafka_cgrp.c

```diff
@@ -5186,7 +5250,6 @@ rd_kafka_cgrp_modify_subscription(rd_kafka_cgrp_t *rkcg,
         revoking = rd_kafka_cgrp_calculate_subscribe_revoking_partitions(
             rkcg, unsubscribing_topics);
 
-        rd_kafka_topic_partition_list_destroy(rkcg->rkcg_subscription);
         rd_kafka_cgrp_subscription_set(rkcg, rktparlist);
```

Comment:
After rebasing, here it should assign the returned value to `cgrp_subscription_version` and pass it to `rd_kafka_cgrp_metadata_refresh`